src/depends.py

```diff
@@ -150,6 +162,8 @@ def detectOSRelease():
                     pass
         if detectOS.result == "Ubuntu" and version < 14:
             detectOS.result = "Ubuntu 12"
+        elif detectOS.result == "Ubuntu" and version == 20.04:
```

Comment:
`version >= 20.04`

Reply:
Also there are no `Ubuntu 20` keys inside the dictionaries listing dependencies.